components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java

```diff
@@ -5929,4 +5943,20 @@ private void setOldOTPRegexValue(
         }
         propertiesFromConnectors.get(otpLengthProperty).setValue(Integer.toString(otpLength));
     }
+
+    private void handlePropertyChanges(Connection dbConnection, int tenantId,
+                                       int idpId, List<IdentityProviderProperty> idpProperties,
+                                       String recoveryNotificationPasswordValue)
+        throws SQLException {
+
+        // Set value of Recovery.Notification.Password.Enable to Recovery.Notification.Password.emailLink.Enable
+        // property to keep backward compatibility. This is only run once per tenant.
+        IdentityProviderProperty property = new IdentityProviderProperty();
+        property.setName("Recovery.Notification.Password.emailLink.Enable");
```

Comment:
Lets have a static constant for this

Reply:
Addressed with https://github.com/wso2/carbon-identity-framework/pull/5601/commits/dc5434b3928be58732d3010cbe33a64acb9825de